supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -212,6 +232,16 @@ Do not use "BIOS" as a generic term to refer to computer firmware. Use "firmware
 
 *See also*: xref:filestore[FileStore], xref:object-store[Object Store]
 
+[[boolean-dependencies]]
+==== image:images/yes.png[yes] Boolean dependencies (noun)
+*Description*: In Red Hat Enterprise Linux, _Boolean dependencies_ are Boolean expressions such as `if`, `and`, `or`, and other expressions that are used in the `Requires`, `Conflicts`, and `Weak` dependency directives. Boolean dependencies are also known as _Rich dependencies_.
```

Comment:
definitely "Rich," capital R? i'm finding both in search results (ofc).

Reply:
Yes, already discussed in other comment above